mscore/prefsdialog.cpp

```diff
@@ -150,10 +150,10 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
       connect(bgColorButton, &QRadioButton::toggled, this, &PreferenceDialog::updateBgView);
 
       zoomDefaultType->clear();
-      zoomDefaultType->addItem(tr("Page Width"), 0);
-      zoomDefaultType->addItem(tr("Whole Page"), 1);
-      zoomDefaultType->addItem(tr("Two Pages"), 2);
-      zoomDefaultType->addItem(tr("Percentage"), 3);
+      zoomDefaultType->addItem(tr("Percentage"), 0);
+      zoomDefaultType->addItem(tr("Page Width"), 1);
+      zoomDefaultType->addItem(tr("Whole Page"), 2);
+      zoomDefaultType->addItem(tr("Two Pages"), 3);
```

Comment:
This may cause issues with translations?

Reply:
Will it? The strings are exactly the same (as is the context) but just reordered for consistency.

Reply:
Not sure

Reply:
My understanding is that Qt uses the literal string and the context together to uniquely identify the translatable string. On Transifex, the filename and line and number are displayed as a convenience, but the identity of the translatable string is not tied to either of those things. (If it were, adding or removing unrelated lines elsewhere in the file would break translations, and we know that that doesn't happen.)

So I'm pretty sure that a simple reordering with no material changes would have no effect on translations.

Reply:
True, so most probably not an issue